src/codecatalyst/auth.ts

```diff
@@ -138,15 +137,6 @@ export class CodeCatalystAuthenticationProvider {
                 codecatalyst_connectionFlow: 'Create',
             } satisfies ConnectionFlowEvent as MetricShapes[MetricName])
 
-            const message = `The ${
-                getIdeProperties().company
-            } Toolkit extension requires a connection for CodeCatalyst to begin.\n\n Proceed to the browser to allow access?`
-
-            const resp = await vscode.window.showInformationMessage(message, { modal: true }, continueItem, cancelItem)
-            if (resp !== continueItem) {
-                throw new ToolkitError('Not connected to CodeCatalyst', { code: 'NoConnection', cancelled: true })
-            }
-
             const newConn = await createBuilderIdConnection(this.auth, defaultScopes)
```

Comment:
This call contains the 2nd prompt and throws if cancelled.

Reply:
Is `promptNotConnected` called anywhere where the first prompt adds context? If so, you could change those callsites to take users to the auth page.